client/src/Backend/GameLogic/GameManager.ts

```diff
@@ -4792,27 +4792,29 @@ class GameManager extends EventEmitter {
    * know the location of either planet. Takes into account links.
    */
   getDist(fromId: LocationId, toId: LocationId): number {
-    const from = this.entityStore.getPlanetWithId(fromId);
-    const to = this.entityStore.getPlanetWithId(toId);
-
-    if (!from) throw new Error('origin planet unknown');
-    if (!to) throw new Error('destination planet unknown');
-    if (!isLocatable(from)) throw new Error('origin location unknown');
-    if (!isLocatable(to)) throw new Error('destination location unknown');
```

Comment:
The `isLocatable` already does a check on `planet !== undefined` here:
https://github.com/dfarchon/DFARES-v0.1/blob/29fdeacd2ef4b86ffc6631366ce4982313a810c1/packages/gamelogic/src/planet.ts#L35-L37